WHITEPAPER.md

```diff
@@ -212,7 +212,7 @@ At this point the broadcaster can begin streaming video segments towards the tra
 3. The protocol can use the next block hash to deterministically select the correct Transcoder for this job.
 4. **Transcoder** -> **Broadcaster**: send output streamID and receipt that the job is accepted.
 5. **Broadcaster** -> **Transcoder**: send stream segments, which contain signatures verifying the input data.
-6. **Broadcaster** -> **Swarm**: Write input data payloads, using SWEAR params to ensure the data will be there long enough for verification (`PersistenceLength` time).
+6. **Broadcaster** -> **Swarm**: Write input data payloads, using SWEAR params to ensure the data will be there long enough for verification (`VerificationPeriod` time).
```

Comment:
Maybe we should generalize this to be any decentralized storage solution with Swarm being one of the options?

Reply:
Yah, I'm going to think about this more. I like that with Swarm we can reference the exact mechanism for storage guarantees, whereas with the others it's more nebulous how that will work. But obviously none are there yet.